src/config/gobjects.rs

```diff
@@ -357,7 +344,7 @@ fn parse_object(
     let ref_mode = toml_object
         .lookup("ref_mode")
         .and_then(Value::as_str)
-        .and_then(ref_mode_from_str);
+        .and_then(|v| FromStr::from_str(v).ok());
```

Comment:
You'd usually use `|v| v.parse().ok()` here. `FromStr` would rarely be called directly

Reply:
(also in the other places in this commit of course)